server/plugins/webapp/register-sw.html

```diff
@@ -0,0 +1,7 @@
+<script>
```

Comment:
for now it makes sense to be inline but before going to production this piece of code will need to handle all cases.

1. What if SW registration fails ?
2. Showing a message like `Caching complete! Future visits will work offline.` when SW is in state  `installed`.